includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -553,9 +553,10 @@ public function javascript_params() {
 		$stripe_params['isAmazonPayAvailable'] = WC_Stripe_Feature_Flags::is_amazon_pay_available();
 
 		// Optimized Checkout feature flag + setting + whether we are on any of the pages that should not show OC.
-		$should_show_optimized_checkout               = $this->oc_enabled && $this->is_valid_optimized_checkout_page();
-		$stripe_params['isOCEnabled']                 = $should_show_optimized_checkout;
-		$stripe_params['shouldShowOptimizedCheckout'] = $should_show_optimized_checkout;
+		$should_show_optimized_checkout                 = $this->oc_enabled && $this->is_valid_optimized_checkout_page();
```

PHPMD (2.15.0)

[error] The variable $should_show_optimized_checkout is not named in camelCase. (undefined)

(CamelCaseVariableName)